assets/templates/aws.billing/schema-b/configs.yml

```diff
@@ -0,0 +1,82 @@
+- name: cloud.region
+  enum: ["us-east-1", "us-east-2", "us-west-1", "us-west-2", "ap-south-1", "ap-northeast-3", "ap-northeast-2", "ap-southeast-1", "ap-southeast-2", "ap-northeast-1", "ca-central-1", "eu-central-1", "eu-west-1", "eu-west-2", "eu-west-3", "eu-north-1", "sa-east-1", "af-south-1", "ap-east-1", "ap-south-2", "ap-southeast-3", "eu-south-2", "eu-central-2", "me-south-1", "me-central-1"]
+  cardinality:
+    numerator: 1
+    denominator: 25
+- name: cloud.account.id
+  value: "123456789"
+- name: cloud.account.name
+  value: sample-account
+- name: aws.billing.currency
+  value: "USD"
+- name: aws.billing.ServiceName
```

Comment:
nit: `aws.billing.ServiceName` can be empty: it means it's reporting `EstimatedCharges` for the whole account.
it should be the sum of all the entries for the same single metrics polling (in terms of metricbeat's period) 
we cannot apply this logic
but we should replicate the fact that there will be one entry for each all the `aws.billing.ServiceName` and one `aws.billing.ServiceName == ""` in the cardinality

Reply:
added `""` as a case in the enum.